__tests__/Description.test.js

```diff
@@ -22,4 +24,37 @@ test('links render in blue', () => {
   const link = renderer.create(<BlueLink />).toJSON();
   expect(link).toMatchSnapshot();
   expect(link).toHaveStyleRule('color', '#008489');
+});
+
+describe('<Modal /> component', () => {
+  it('should change state when modal is closed', () => {
+    const wrapper = shallow(<Description />);
+    const modal1 = wrapper.find('button').at(0);
+    const modal2 = wrapper.find('button').at(1);
+
+    modal1.simulate('click');
+    expect(wrapper.state('modal1IsOpen')).toEqual(false);
+
+    modal2.simulate('click');
+    expect(wrapper.state('modal1IsOpen')).toEqual(false);
```

Comment:
Not sure if intentional but you check for `'modal1IsOpen'` on both line 39 and 36.

Reply:
@AndrewPomo good catch - updated to modal2!